docs/resources/apig_api.md

```diff
@@ -338,35 +356,35 @@ The `web_policy` block supports:
 <a name="apig_api_conditions"></a>
 The `conditions` block supports:
 
-* `value` - (Required, String) Specifies the condition type. For a condition with the input parameter source:
+* `value` - (Required, String) Specifies the value of the backend policy.  
+  For a condition with the input parameter source:
   + If the condition type is **Enumerated**, separate condition values with commas.
   + If the condition type is **Matching**, enter a regular expression compatible with PERL.
 
   For a condition with the Source IP address source, enter IPv4 addresses and separate them with commas. The CIDR
   address format is supported.
 
-* `param_name` - (Optional, String) Specifies the request parameter name. This parameter is required if the policy type
-  is param.
+* `param_name` - (Optional, String) Specifies the request parameter name.
+  This parameter is required if the policy type is **param**.
 
-* `source` - (Optional, String) Specifies the policy type. The valid values are **param** and **source**, default to
-  **source**.
+* `source` - (Optional, String) Specifies the backend policy type.  
+  The valid values are **param** and **source**, defaults to **source**.
 
-* `type` - (Optional, String) Specifies the condition type of the backend policy. The valid values are **Equal**,
-  **Enumerated** and **Matching**, default to **Equal**.
+* `type` - (Optional, String) Specifies the condition type of the backend policy.  
+  The valid values are **Equal**, **Enumerated** and **Matching**, defaults to **Equal**.
 
 ## Attributes Reference
 
 In addition to all arguments above, the following attributes are exported:
 
-* `id` - ID of the APIG API.
-* `register_time` - Time when the API is registered, in UTC format.
-* `update_time` - Time when the API was last modified, in UTC format.
+* `id` - The API ID.
+* `registered_at` - The registered time of the API.
+* `updated_at` - The latest update time of the API.
 
 ## Import
 
-APIs can be imported using their `name` and ID of the APIG dedicated instance to which the API belongs, separated by a
-slash, e.g.
+APIs can be imported using their `name` and the related dedicated instance IDs, separated by a slash, e.g.
```

Comment:
keep same order with import command

Reply:
the command is clear and I don't think we should reorder it.